l2geth/rollup/sync_service.go

```diff
@@ -172,19 +172,21 @@ func NewSyncService(ctx context.Context, cfg Config, txpool *core.TxPool, bc *co
 			}
 		}
 
-		// Wait until the remote service is done syncing
-		tStatus := time.NewTicker(10 * time.Second)
-		for ; true; <-tStatus.C {
-			status, err := service.client.SyncStatus(service.backend)
-			if err != nil {
-				log.Error("Cannot get sync status")
-				continue
-			}
-			if !status.Syncing {
-				tStatus.Stop()
-				break
+		if !cfg.IsVerifier {
```

Comment:
This will only run for the sequencer which I'm not sure is solving the problem. It's the replicas that we want to speed up, I think it should be safe to skip waiting if it's a verifier and backend L2. We would need to test running a verifier with backend L1 before I felt safe merging it